pkg/operatorstatus/operatorstatus.go

```diff
@@ -59,40 +59,47 @@ type ClusterOperatorWrapper interface {
 }
 
 func (o *ClusterOperatorHandler) UpdateOperatorStatus(sampleResource *v1alpha1.SamplesResource) error {
+
 	var err error
+
 	switch {
 	case sampleResource.ConditionTrue(v1alpha1.SamplesExist):
-		err = o.setOperatorStatus(configv1.OperatorAvailable, configv1.ConditionTrue, "Samples exist in the openshift project")
+		err = o.setOperatorStatus(configv1.OperatorAvailable, configv1.ConditionTrue, fmt.Sprintf("Samples installed at %s", v1alpha1.CodeLevel))
 	case sampleResource.ConditionFalse(v1alpha1.SamplesExist):
-		err = o.setOperatorStatus(configv1.OperatorAvailable, configv1.ConditionFalse, "Samples do not exist in the openshift project")
+	case sampleResource.ConditionUnknown(v1alpha1.SamplesExist):
+		err = o.setOperatorStatus(configv1.OperatorAvailable, configv1.ConditionFalse, fmt.Sprintf("Samples not installed at %s", v1alpha1.CodeLevel))
 	default:
-		err = o.setOperatorStatus(configv1.OperatorAvailable, configv1.ConditionUnknown, "The presence of the samples in the openshift project is unknown")
+		err = o.setOperatorStatus(configv1.OperatorAvailable, configv1.ConditionUnknown, fmt.Sprintf("Samples at %s unknown", v1alpha1.CodeLevel))
```

Comment:
i don't love this one but i have nothing better to suggest at the moment.

Reply:
Yeah in fact in the changes I noted in https://github.com/openshift/cluster-samples-operator/pull/70#issuecomment-448399599 I got rid of this ... but we can certainly revisit when I push the new commit